Mathlib/Order/KrullDimension.lean

```diff
@@ -860,6 +864,8 @@ lemma KrullDimLE.mono {n m : ℕ} (e : n ≤ m) (α : Type*) [Preorder α] [Krul
     KrullDimLE m α :=
   ⟨KrullDimLE.krullDim_le (n := n).trans (Nat.cast_le.mpr e)⟩
 
+instance {α} [Preorder α] [Subsingleton α] : KrullDimLE 0 α := ⟨krullDim_nonpos_of_subsingleton⟩
```

Comment:
Probably you can prove it for all `n` other than `0`. Do we care?

Reply:
generalized

Reply:
I think we should rather add an `instance [krullDimLE 0 α] : krullDimLE n α` using [Order.KrullDimLE.mono](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Order/KrullDimension.html#Order.KrullDimLE.mono)

Reply:
Sure.